scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/HttpTransferProcessor.java

```diff
@@ -54,19 +61,26 @@ public class HttpTransferProcessor extends TimeBasedProcessor {
           + "}";
 
   private final int numAccounts;
-  private final boolean isVerification;
-  private final boolean useCompactLog;
```

Comment:
Remove the verification test things because we don't run verification test using this processor.